### PR TITLE
Aspect block item pairs

### PR DESCRIPTION
**Pull request by glubtier**
- Migrated Aspect tree blocks to ItemBlockPairs and updated related references
- Signs were not migrated, since they are not one block to one item
- Added signs to MINEABLE_WITH_AXE tag (not added to changelog as there is no 1.21 changelog yet.)

### CODE REVIEW

**Comment by glubtier** [comment]
@kirderf1 Wanted to check in where this is at. Know everyone's busy, just didn't want it to get lost, and I'm waiting for it to be done so I can move onto moons.

**Comment by glubtier** [comment]
Updated for 1.21.1

File: src/main/java/com/mraof/minestuck/data/AspectTreeBlocksData.java [hunk 1]
```diff
@@ -21,845 +21,841 @@ public final class AspectTreeBlocksData
...
-		provider.addBlock(BLOOD_ASPECT_SLAB, "Blood Slab");
-		provider.addBlock(BLOOD_ASPECT_FENCE, "Blood Fence");
-		provider.addBlock(BLOOD_ASPECT_FENCE_GATE, "Blood Fence Gate");
-		provider.addBlock(BLOOD_ASPECT_DOOR, "Blood Door");
-		provider.addBlock(BLOOD_ASPECT_TRAPDOOR, "Blood Trapdoor");
-		provider.addBlock(BLOOD_ASPECT_PRESSURE_PLATE, "Blood Pressure Plate");
-		provider.addBlock(BLOOD_ASPECT_BUTTON, "Blood Button");
-		provider.addBlock(BLOOD_ASPECT_BOOKSHELF, "Blood Bookshelf");
-		provider.addBlock(BLOOD_ASPECT_LADDER, "Blood Ladder");
+		provider.addBlock(BLOOD_ASPECT_LOG.blockHolder(), "Blood Log");
```
Comment (kirderf1) [inline src/main/java/com/mraof/minestuck/data/AspectTreeBlocksData.java:24]: `MinestuckLanguageProvider` has an overload for ItemBlockPair. You can therefore write this as `provider.add(BLOOD_ASPECT_LOG, "Blood Log");` (you can see this used in `SkaiaBlocksData`)

**Review by kirderf1 — COMMENTED** [review]
Now that this is up to date with the main branch, you should add any data changes/fixes to the changelog.

**Comment by glubtier** [comment]
> Now that this is up to date with the main branch, you should add any data changes/fixes to the changelog.

Fixes done, I added making signs mineable with axes to the changelog but I don't think the rest is relevant.

**Review by kirderf1 — APPROVED** [review]

**Pull request merged**